static/app/views/detectors/components/forms/metric.tsx

```diff
@@ -0,0 +1,642 @@
+          size="md"
+          style={{width: 'min-content'}}
+          priority="primary"
+          icon={<IconAdd />}
+        >
+          Connect Automations
+        </Button>
+      </Section>
+    </Container>
+  );
+}
+
+function AssignSection() {
+  return (
+    <Container>
+      <Section title={t('Assign')}>
+        <OwnerField />
+      </Section>
+    </Container>
+  );
+}
+
+function PrioritizeSection() {
+  const kind = useFormField<MetricDetectorKind>('kind')!;
+  return (
+    <Container>
+      <Section
+        title={t('Prioritize')}
+        description={
+          kind === 'dynamic'
+            ? t('Sentry will automatically update priority.')
+            : t('Update issue priority when the following thresholds are met:')
+        }
+      >
+        {kind !== 'dynamic' && <PriorityControl name="prioritize" />}
+      </Section>
+    </Container>
+  );
+}
+
+function DetectSection() {
+  const kind = useFormField<MetricDetectorKind>('kind')!;
+  const aggregateOptions: Array<[string, string]> = useMemo(() => {
+    return ALLOWED_EXPLORE_VISUALIZE_AGGREGATES.map(aggregate => {
+      return [aggregate, aggregate];
+    });
+  }, []);
+
+  return (
+    <Container>
+      <Section
+        title={t('Detect')}
+        description={t('Sentry will check the following query:')}
+      >
+        <FirstRow>
+          <DetectColumn style={{maxWidth: 500}} gap={space(1)}>
+            <VisualizeField
+              placeholder={t('Metric')}
+              flexibleControlStateSize
+              inline={false}
+              label="Visualize"
+              name="visualize"
+              choices={[['span.duration', 'span.duration']]}
+              defaultValue="span.duration"
+            />
+            <AggregateField
+              placeholder={t('aggregate')}
+              flexibleControlStateSize
+              defaultValue={'p75'}
+              name="aggregate"
+              choices={aggregateOptions}
+            />
+          </DetectColumn>
+          <DetectColumn>
+            <FilterField />
+          </DetectColumn>
+        </FirstRow>
+        <MonitorKindField
+          label="...and monitor for changes in the following way:"
+          flexibleControlStateSize
+          inline={false}
+          name="kind"
+          defaultValue="threshold"
+          choices={[
+            [
+              'threshold',
+              t('Threshold'),
+              t('Absolute-valued thresholds, for non-seasonal data.'),
+            ],
+            ['change', t('Change'), t('Percentage changes over defined time windows.')],
+            [
+              'dynamic',
+              t('Dynamic'),
+              t('Auto-detect anomalies and mean deviation, for seasonal/noisy data.'),
+            ],
+          ]}
+        />
+        <Flex column>
+          {(!kind || kind === 'threshold') && (
+            <Flex column>
+              <MutedText>An issue will be created when query value exceeds:</MutedText>
```

Comment:
translations but i think its probably fine for now

Reply:
translated in 92ed66c8014 🌐